DigitalExcellence/dex-frontend
# Feature/362 implement new project detail

## Description

Implement the modal-design of the project detail page

## Type of change


## Checklist


## Steps to Test or Reproduce

Outline the steps to test or reproduce the PR here.
These steps will be used during release testing.

1. Go to the project overview page and click a project
2. A modal should pop up with the project details

## Link to issue
#362 

Closes: #362
